File: week1/pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height, spaces, hash;
    
    //user's input
    do 
    {
        printf("Height:");
        height = GetInt();
    }
    while (height <= 0 || height >= 23);
    
    //rows loop
    for (int a = 0; a < height; a++)
    {
        
    //spaces/hashes loop
        for (spaces = height - a; spaces >= 0; spaces--)
        {
            printf(" ");
        }
        for (hash = 0; hash <= (a); hash++)
        {
            printf("#");    
        }
    printf("#\n");
    }
}

File: week8/pset6/mario.py
import cs50

while True:
    print("Type a number between 0 and 23: ", end="")
    height = cs50.get_int()
    if height > 0 or height < 23:
        break

a = 0
for a  in range(a,height,+1):
    spaces = height - a
    for spaces  in range(spaces,0,-1):
        print(" ", end="")
    spaces = spaces - 1
    print("#" * (a+2), end="")
    print()
